src/awx_plugins/credentials/plugins.py

```diff
@@ -8,6 +8,7 @@
     gettext_noop,
 )
 
+from ._managed_types.terraform import hcp_terraform
```

Comment:
Instead of re-exporting, point the entry points in the core packaging metadata (configured though `pyproject.toml` via PEP 621) to the new module directly. There's no reason to expose these objects here as well as this module isn't supposed to be importable directly, outside of the plugin loading mechanism.